configurator/pom.xml

```diff
@@ -74,6 +74,9 @@
                         <Provide-Capability>
                             osgi.extender;osgi.extender="osgi.configurator";version:Version="1.0"
                         </Provide-Capability>
+                        <Require-Capability>
+                            osgi.implementation;filter:='(&(osgi.implementation=osgi.cm)(version>=1.6.0)(!(version>=2.0.0)))'
```

Comment:
You need to encode entities in XML!

--> `osgi.implementation;filter:='(&amp;(osgi.implementation=osgi.cm)(version>=1.6.0)(!(version>=2.0.0)))'`

Reply:
I think that would be:
`osgi.implementation;filter:='(&amp;(osgi.implementation=osgi.cm)(version&gt;=1.6.0)(!(version&gt;=2.0.0)))'`

Perhaps better to use an annotation in the source for safety?

Reply:
This project still uses the bundle plugin. I wasn't sure how recent the BND implementation behind it is. Thus I didn't use the Annotation. Would be preferable though.